proxy/src/http/project.rs

```diff
@@ -184,11 +184,6 @@ mod handler {
                 shareable_entity_identifier,
                 metadata: meta.into(),
                 registration: None,
-                stats: project::Stats {
```

Comment:
So judging from these, maybe we made the wrong call by _adding_ and endpoint for `get_stats` here. Maybe we should actually have added the call to `browser.get_stats` where we're fetching a project. Which makes sense really. Sorry, this was my bad :hand: